packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue

```diff
@@ -30,28 +30,28 @@
 </template>
 
 <script setup lang="ts">
-import { ModelConfiguration } from '@/types/Types';
-import {
-	getInitialName,
-	getInitialDescription,
-	getInitialUnit,
-	getInitialExpression,
-	getInitialSource
-} from '@/services/model-configurations';
+import { Model, ModelConfiguration } from '@/types/Types';
+import { getInitialExpression, getInitialSource } from '@/services/model-configurations';
 import TeraInput from '@/components/widgets/tera-input.vue';
 import { ref } from 'vue';
 import Button from 'primevue/button';
+import {
+	getInitialDescription,
+	getInitialName,
+	getInitialUnits
+} from '@/model-representation/service';
 
 const props = defineProps<{
+	model: Model;
 	modelConfiguration: ModelConfiguration;
```

Comment:
why do things like initial name, unit, description all come from model
and initial source, and expression comes from the configuration?

Can each configuration have its own unique source but not its own unique description?

Reply:
thats correct they cannot have unique names, units, descriptions so those come from Model. but can have unique sources and expressions 😄